app/models/order.rb

```diff
@@ -1,7 +1,7 @@
 class Order < ApplicationRecord
   belongs_to :employee
   belongs_to :reward
-  enum status: { placed: 'placed', delivered: 'delivered', not_set: 'not_set' }, _suffix: true
+  enum status: { placed: 0, delivered: 1 }, _suffix: true
```

Comment:
do we really need that suffix here? with suffix the helpers will work like `Order.delivered_status?` without it will look like: `Order.delivered?` I prefer the latter option, since it seems for me easier to read. But, as always it is a matter of preference :)